net-libs/libircclient/libircclient-1.8.recipe

```diff
@@ -1,20 +1,19 @@
 SUMMARY="An IRC client library"
-DESCRIPTION="
-libircclient is a library implementing the client-server IRC protocol.
-"
+DESCRIPTION="libircclient is a library implementing the client-server IRC protocol. \
+It is designed to be small, fast, portable and compatible with the RFC standards \
+as well as non-standard but popular features. \
```

Comment:
Remove the " \" after "features.". This is supposed to be a paragraph, i.e. a newline is intended.